newIDE/app/src/InstancesEditor/HighlightedInstance.js

```diff
@@ -47,5 +56,37 @@ export default class InstancesSelection {
       highlightRectangle.height
     );
     this.highlightRectangle.endFill();
+
+    const tooltipInfo =
+      'Object: ' +
+      this.highlightedInstance.getObjectName() +
+      '\n' +
+      'X: ' +
+      parseInt(this.highlightedInstance.getX()) +
+      '  Y: ' +
+      parseInt(this.highlightedInstance.getY()) +
+      '\n' +
+      'Layer: ' +
+      this.highlightedInstance.getLayer() +
+      '  Z: ' +
+      this.highlightedInstance.getZOrder() +
+      '\n';
+    this.tooltip.text = tooltipInfo;
+    this.tooltip.x = //alternatively we could draw it at mouse position instead. Would help for big objects
+      highlightRectangle.x -
+      this.tooltip.width / 2 +
+      highlightRectangle.width / 2;
+    this.tooltip.y = highlightRectangle.y - 68;
+
+    this.tooltipBG.clear();
+    this.tooltipBG.beginFill(0x000000, 0.8);
+    this.tooltipBG.drawRoundedRect(
+      this.tooltip.x - 5,
+      this.tooltip.y - 5,
+      this.tooltip.width + 10,
+      this.tooltip.height - 5,
+      16
```

Comment:
16 is really huge in terms of rounding radius ^^
Let's go for something more subtle.

macOS tooltips are not rounded but have a slight shadow:
![image](https://user-images.githubusercontent.com/1280130/56215607-d5638080-6057-11e9-974a-acb9bc232bfd.png)

Nothing special for Windows 10, just a gray border:
![image](https://user-images.githubusercontent.com/1280130/56215706-fdeb7a80-6057-11e9-9e7e-991cc226f98a.png)

Reply:
how about 4 :)

Reply:
with 4, I can remove { antialias: true } from the renderer- as jaggy edges are no longer noticeable

Reply:
Actually,lets keep it. It does look a bit ugly sometimes it draws it